mergify_engine/web/api/queues.py

```diff
@@ -317,5 +324,406 @@ async def repository_queues(
                 )
 
             queues.queues.append(queue)
-
         return queues
+
+
+QueuePauseSchema = voluptuous.Schema({voluptuous.Required("reason"): str})
+
+
+@router.post(
+    "/repos/{owner}/{repository}/queue/{queue_name}/pause-from",  # noqa: FS003
+    summary="Pauses merge queue(s)",
+    description="Pauses the merge of a queue and the queues following it",
+    responses={
+        **api.default_responses,  # type: ignore
+        404: {"description": "Not found"},
+        200: {"description": "The queue(s) have been paused"},
+    },
+)
+async def pause_queues_from(
+    request: requests.Request,
+    application: application_mod.Application = fastapi.Depends(  # noqa: B008
+        security.get_application
+    ),
+    owner: github_types.GitHubLogin = fastapi.Path(  # noqa: B008
+        ..., description="The owner of the repository"
+    ),
+    repository: github_types.GitHubRepositoryName = fastapi.Path(  # noqa: B008
+        ..., description="The name of the repository"
+    ),
+    queue_name: rules.QueueName = fastapi.Path(  # noqa: B008
+        ..., description="The name of the queue"
+    ),
+    redis_cache: utils.RedisCache = fastapi.Depends(  # noqa: B008
+        redis.get_redis_cache
+    ),
+    redis_queue: utils.RedisQueue = fastapi.Depends(  # noqa: B008
+        redis.get_redis_queue
+    ),
+    installation_json: github_types.GitHubInstallation = fastapi.Depends(  # noqa: B008
+        security.get_installation
+    ),
+) -> responses.JSONResponse:
+
+    try:
+        raw_json = await request.json()
+        data = QueuePauseSchema(raw_json)
+
+    except voluptuous.Invalid:
+        return responses.JSONResponse(
+            status_code=400, content={"error": "Invalid json"}
```

Comment:
You need to pass the details from voluptuous so the user understand what's wrong.